ste/uploader-blobFS.go

```diff
@@ -169,22 +175,44 @@ func (u *blobFSUploader) GenerateUploadFunc(id common.ChunkID, blockIndex int32,
 	})
 }
 
+var flushThreshold int64 = (1024 * 1024 * 1024) * 20 //20GB
+//TODO: Should a flag be present for the threshold?
+var thresholdSetup bool
+
+func setupThreshold(blockSize int64) {
+	//Find the closest multiple of the block size to the flush threshold
+	m := flushThreshold % blockSize
+
+	if math.Abs(float64(m)) < math.Abs(float64((flushThreshold-m+blockSize)-flushThreshold)) {
```

Comment:
This line looks odd. Is it just saying if m < blocksize/2?

Reply:
Actually... That's a really good point and I don't know what I was thinking.

Reply:
I was trying to find which block the flushThreshold was closest to, but, you just pointed out a way better solution.